packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -4756,161 +5146,187 @@ curl -v -X POST \
+            "firstName":"First",
+            "lastName":"Last",
+            "locale":"en",
+            "timeZone":"America/Los_Angeles"
+         }
+      },
+      "factor":{
+         "id":"fwfb2yngen2hwcp5z0g4",
+         "factorType":"webauthn",
+         "provider":"FIDO",
+         "vendorName":"FIDO",
+         "profile":{
+            "credentialId":"Ab1jRznBN0A1JAz_xwdZEW6yDqGdXUfO-GWhmzLr2klR1VjdpHzYjuA_h5QM1BBWqpTYuRT6isPphfH9F8HaRAZQ",
+            "authenticatorName":"MacBook Touch ID"
+         },
+         "_embedded":{
+            "challenge":{
+               "challenge":"K0UNqWlz2TCCDd5qEkBf",
+               "extensions":{
+               }
+            }
+         }
+      },
+      "policy":{
+         "allowRememberDevice":false,
+         "rememberDeviceLifetimeInMinutes":0,
+         "rememberDeviceByDefault":false,
+         "factorsPolicyInfo":{
+
+         }
       }
-    },
-    "prev": {
-      "href": "https://${yourOktaDomain}/api/v1/authn/previous",
-      "hints": {
-        "allow": [
-          "POST"
-        ]
+   },
+   "_links":{
+      "next":{
+         "name":"verify",
+         "href":"https://${yourOktaDomain}/api/v1/authn/factors/fwfb2yngen2hwcp5z0g4/verify",
+         "hints":{
+            "allow":[
+               "POST"
+            ]
+         }
+      },
+      "prev":{
+         "href":"https://${yourOktaDomain}/api/v1/authn/previous",
+         "hints":{
+            "allow":[
+               "POST"
+            ]
+         }
+      },
+      "cancel":{
+         "href":"https://${yourOktaDomain}/api/v1/authn/cancel",
+         "hints":{
+            "allow":[
+               "POST"
+            ]
+         }
       }
-    }
-  }
+   }
 }
 ```
 
-##### Verify Call Challenge (OTP)
+##### Get the Signed Assertion from the WebAuthn Authenticator
 
-Specify `passCode` in the request to verify the factor.
 
-###### Request Example for Verify Call Challenge
+```html
+<!-- Using CryptoUtil.js from https://github.com/okta/okta-signin-widget/blob/master/src/util/CryptoUtil.js -->
+<script>
+  // Convert activation object's challenge nonce from string to binary
+  factor._embedded.challenge.challenge = CryptoUtil.strToBin(factor._embedded.challenge.challenge); 
+
+  // Call the WebAuthn javascript API to get signed assertion from the WebAuthn authenticator
+  navigator.credentials.get({
+    publicKey: factor._embedded.challenge
+  })
+    .then(function (assertion) {
+      // Get the client data, authenticator data, and signature data from callback result, convert from binary to string
+      var clientData = CryptoUtil.binToStr(assertion.response.clientDataJSON);
+      var authenticatorData = CryptoUtil.binToStr(assertion.response.authenticatorData);
+      var signatureData = CryptoUtil.binToStr(assertion.response.signature);
+    })
+    .fail(function (error) {
+      // Error from WebAuthn platform
+    });
+</script>
+```
+
+##### Post the Signed Assertion to Okta to Complete Verification
```

Comment:
sentence case for headings

Post the signed assertion to Okta to complete verification